yarn/Dockerfile

```diff
@@ -1,16 +1,16 @@
 # Use the base App Engine Docker image, based on debian jessie.
 FROM launcher.gcr.io/google/debian8
 
-# https://yarnpkg.com/en/docs/install#linux-tab
-RUN apt-get update && \
-    apt-get install -y curl apt-transport-https git bzip2 build-essential && \
+# Install updates and dependencies
+RUN apt-get update -y && apt-get install --no-install-recommends -y -q curl python build-essential git ca-certificates libkrb5-dev imagemagick apt-transport-https && \
+    apt-get clean && rm /var/lib/apt/lists/*_*
 
-    # Install newer Node version.
-    curl -sL https://deb.nodesource.com/setup_8.x | bash - && \
-    apt-get install -y nodejs && \
+ARG NODE_VERSION
+RUN mkdir /nodejs && curl https://nodejs.org/dist/${NODE_VERSION}/node-${NODE_VERSION}-linux-x64.tar.gz | tar xvzf - -C /nodejs --strip-components=1
+ENV PATH $PATH:/nodejs/bin
 
-    # Install Yarn
-    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+# Install Yarn
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
```

Comment:
Is the Yarn version important?

Reply:
I think so. But I'm a little conflicted how to handle it given that the npm cloud builder is tagged based on the node version and not the NPM version.

Reply:
That said - I think most people use the latest yarn stable and would care mostly about the node version.

Reply:
I agree with you on that.

If so, could we have the same logic as for the npm builder which builds many versions of nodejs? https://github.com/GoogleCloudPlatform/cloud-builders/blob/master/npm/cloudbuild.yaml#L5

Reply:
Yeah sounds good - I'll amend this PR with that.

Reply:
Awesome, thank you so much @palmerj3

Reply:
@Philmod I just updated the PR. I tested everything locally.